src/components/DashboardAppBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const DashboardAppBar = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const logout = () => {
+    firebase.logout().then(() => {
+      navigate('/')
+    })
+  }
+  return (
+    <div>
+      <AppBar position="static">
+        <Toolbar>
+          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
```

Comment:
Oh no, why `component="div"`? Typography components should be one of the text tags (like `h6`, `span`, `p` etc.). I know that this is possible to set it to `div`, but it is usually possible to keep our HTML in more [semantic](https://www.freecodecamp.org/news/semantic-html5-elements/) way.

Also, maybe it is just copy-paste, but note that typically using only `variant` is enough. `variant` sets related styles and also uses picked variant as HTML tag. So, you eventually would need to use `component` when you will need to use style for one tag (`h6`, for example), but override actual tag (to `p` or anything else).